src/server/utilities/cspHeader/index.js

```diff
@@ -114,6 +114,7 @@ const directives = {
       'https://www.youtube.com', // Social Embeds, <amp-youtube />
       'https://www.instagram.com', // Social Embeds, <amp-instagram />
       'https://*.ampproject.net', // Social Embeds
+      'https://news.test.files.bbci.co.uk',
```

Comment:
I don't think we need this specified for amp this isn't shown as blocked on https://www.test.bbc.com/gahuza/23323415.amp

Reply:
I'm seeing an error on amp: 
![image](https://user-images.githubusercontent.com/9645462/87278994-d34bd400-c4dd-11ea-9435-26486d4c73bc.png)
The placeholder images are loading in place of the includes though:
![image](https://user-images.githubusercontent.com/9645462/87279027-f37b9300-c4dd-11ea-9169-e3fe9f87ff78.png)

Reply:
The placeholder image behaviour is expected but may give the impression the includes were loading correctly when they were not. I think we do need `https://news.test.files.bbci.co.uk` in the amp csp unless I'm missing something @rhenshaw56 ?

Reply:
yeah, I get csp errors now. 

![Screen Shot 2020-07-13 at 09 27 24](https://user-images.githubusercontent.com/25608386/87281729-6ccdb280-c4eb-11ea-9056-52560386eb16.png)